controllers/goharbor/harbor/resources.go

```diff
@@ -90,6 +90,16 @@ func (r *Reconciler) AddResources(ctx context.Context, resource resources.Resour
 		return errors.Wrapf(err, "cannot add %s", controllers.ChartMuseum)
 	}
 
+	exporterCertificate, err := r.AddExporterConfigurations(ctx, harbor, internalTLSIssuer)
+	if err != nil {
+		return errors.Wrapf(err, "cannot add %s configuration", controllers.Exporter)
```

Comment:
I suggest to no more use sentences in error wrapping.
On new feature that I pushed, I am adding only context without the _negative_ word.
Example:

```suggestion
		return errors.Wrapf(err, "add %s configuration", controllers.Exporter)
```

This way, errors serialisation is shorter.
WDYT?

PS: I know that I was the one who wrote it first, but I now think that I was wrong ☺️

Reply:
Actually, that's a better way for an error statement. I just have one minor concern, should that end with an `error` word?

```go
return errors.Wrapf(err, "add %s configuration error", controllers.Exporter)
```

Reply:
Not sure about the `error` keyword.
The value is already `error` typed.
the `wrap()` feature concatenate strings, so it is not necessary to have `error`:
1. ```
   add resources: add exporter configuration: get certificate
   ```
2. ```
   add resources error: add exporter configuration error: get certificate error
   ```
IMO 1 is more readable.